VAMobile/src/screens/BenefitsScreen/ClaimsScreen/ClaimsHistoryScreen/ClaimsHistoryScreen.tsx

```diff
@@ -48,11 +48,24 @@ function ClaimsHistoryScreen({ navigation }: IClaimsHistoryScreen) {
     data: claimsAndAppealsListPayload,
     isError: claimsAndAppealsListError,
     isLoading: loadingClaimsAndAppealsList,
+    isFetched: finishedLoadingClaimsAndAppeals,
     refetch: refetchClaimsAndAppealsList,
   } = useClaimsAndAppeals(claimType, 1, {
     enabled: claimsAndAppealsAccess && (claimsNotInDowntime || appealsNotInDowntime),
   })
 
+  useEffect(() => {
+    if (finishedLoadingClaimsAndAppeals) {
+      const claimsList = claimsAndAppealsListPayload?.data
+      logAnalyticsEvent(
+        Events.vama_claim_count(
+          claimType === ClaimTypeConstants.CLOSED ? claimsList?.length || 0 : 0,
+          claimType === ClaimTypeConstants.ACTIVE ? claimsList?.length || 0 : 0,
```

Comment:
...this doesn't feel right? These would log the same numbers?

Reply:
nope, we no longer prefetch both active and closed claims, we only fetch what is needed which is the active ones initially. So to keep this same analytic we can only log one or the other. I do think it would be potentially better to log different analytics for active and closed going forward. Just not sure who to run this by.